source/install/troubleshooting.rst

```diff
@@ -63,7 +63,7 @@ If email sign-in was turned off before the System Administrator switched sign-in
 Unable to play audio or video files on browser
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Your server may be misconfigured where the MIME type isn't set correctly. Try installing a media player or read the `related documentation <https://golang.org/pkg/mime/#TypeByExtension>`_ to add the missing MIME types.
+The server might not be using the correct MIME type for the media files. To configure MIME types on your server, see `related documentation <https://golang.org/pkg/mime/#TypeByExtension>`_.
 
```

Comment:
@JeffSchering  Following our discussion yesterday, propose removing this section from install guides, given

 - it should only affects Windows servers, which we don't officially support
 - no one has actually reported the issue yet
 - we're not certain what exact steps are needed to troubleshoot and resolve the issue